buildenv/jenkins/variables/defaults.yml

```diff
@@ -103,15 +103,15 @@ test_dependencies_job_name: 'test.getDependency'
 slack_channel: '#jenkins'
 build_discarder:
   logs:
-    Nightly: 40
-    Release: 5
-    OMR: 15
-    OpenJDK8: 5
-    OpenJDK11: 5
-    OpenJDK17: 10
-    OpenJDK21: 10
-    OpenJDK24: 10
-    OpenJDK: 10
+    Nightly: 30
```

Comment:
You're also setting up for 30 nightlies but they are being cleaned up sooner by the 21 day limit above.

Reply:
I think it's fine. It's possible for multiple nightlies to run manually run in a day.